assets/vue/components/import-modal.vue

```diff
@@ -211,13 +214,18 @@
       editTemplate: function () {
         var editor = this.getEditor()
         var pageId = this.getPageId()
+	    let customizerRedirect = this.siteData.edit_content_redirect;
+        console.log( customizerRedirect );
```

Comment:
@stefan-cotitosu you forgot a console.log here

Reply:
Thank 🙂